.yamato/update_from_upstream.yml

```diff
@@ -0,0 +1,19 @@
+name: Update from upstream
+agent:
+  type: Unity::VM
+  image: platform-foundation/linux-ubuntu18.04-il2cpp-bokken:latest
+  flavor: b1.xlarge
```

Comment:
I'm not sure it matters from resource point of view, but do we need xlarge?

Reply:
I'm not sure about that - I copied what some other jobs in IL2CPP used. I say we stick with this for now and see how things go in the future.